Core/Schemas/SchemaRepo.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.IO;
+using System.Reflection;
+
+namespace Core.Schemas
+{
+    /// <summary>
+    /// Provides easy access to hard-coded schemas that can be used by generators or parsers to inject additional schema
+    /// components using Bebop files.
+    /// </summary>
+    static class SchemaRepo
+    {
+        public static readonly string RpcDatagram;
+
+        static SchemaRepo()
+        {
+            var asm = Assembly.GetExecutingAssembly();
+
+            using (var rsrcStream =
+                   asm.GetManifestResourceStream("Core.Schemas.RpcDatagram.bop"))
+            {
+                using (var sRdr = new StreamReader(rsrcStream))
```

Comment:
I would address the compiler warning here and check if `rsrcStream == null`. I guess there isn't much to do except throw though 😅

Reply:
If it is null that means there is something fundamentally wrong with the binary since it is getting compiled in.